### DISCUSSION TITLE
Improvements to the search

### DISCUSSION THREAD

**Discussion by opidopiopi** [outline/outline#7202] [General]
Hi!

we want to start using Outline for our company's internal wiki. During testing we found some inconveniences with the search.

First of all, the highlight in the search result list itself is quite hard to see. Especially when using dark mode. Maybe it would be possible to not only print the matching string in bold but also put in a different background color?

After executing a search query and opening the resulting document page, the search result is not highlighted. That makes it difficult to quickly find the content looked for. The ideal would be if the matching words of the search query were highlighted on the document page.

Combined with the [issue](https://github.com/outline/outline/discussions/1549) that the user is always brought to the beginning of the document page, search results (may) appear out of sight and not highlighted.

Do you think it is possible to improve on these points?

**Comment by tommoor** [comment]
The contrast of search results can certainly be improved in dark mode, will make some tweaks here.

Highlighting the search result or jumping to it in the page is more troublesome, I don't know of any other products that do this and I think it would be generally unexpected default behavior. It is annoying to have to type out the search again in cmd+f though and many won't know to do that… perhaps there is a button that shows up when clicking through to the search result to let you quickly apply the search to the page.

**Reply by opidopiopi** [reply]
Thanks for the quick answer!
I understand it's not as easy as it sounds. We are grateful for any improvements!
